ee/server/service/teams.go

```diff
@@ -209,6 +209,15 @@ func (svc *Service) ModifyTeam(ctx context.Context, teamID uint, payload fleet.T
 
 		team.Config.Integrations.Jira = payload.Integrations.Jira
 		team.Config.Integrations.Zendesk = payload.Integrations.Zendesk
+		// Only update the google calendar integration if it's not nil
+		if payload.Integrations.GoogleCalendar != nil {
+			invalid := &fleet.InvalidArgumentError{}
+			_ = svc.validateTeamCalendarIntegrations(ctx, team, payload.Integrations.GoogleCalendar, appCfg, invalid)
```

Comment:
ignoring error here since this error will be gone once policy ids move out of config and into policy table